.github/workflows/python.yml

```
@@ -25,8 +25,8 @@ jobs:
           - "core"
           - "jax"
           - "pymc3"
-          # - "pymc4"
-          # - "pymc4_jax"
+          # - "pymc"
+          # - "pymc_jax"
```

These were commented out because the PyMC4 version was broken. Can you uncomment them?
 
```suggestion
          - "pymc"
          - "pymc_jax"
```